modules/iothub/src/iothub.c

```diff
@@ -686,6 +701,145 @@ static IOTHUB_MESSAGE_HANDLE IoTHubMessage_CreateFromGWMessage(MESSAGE_HANDLE me
     return result;
 }
 
+static void SendEventAsync_receiveMessageConfirmation(IOTHUB_CLIENT_CONFIRMATION_RESULT result, void* userContextCallback)
+{
+    if (!userContextCallback)
+    {
+        LogError("Context was null");
+        return;
```

Comment:
>return [](start = 8, length = 6)

The coding standards for our C code is to have one return call.

Reply:
fixed